app/views/kaminari/_next_page.html.erb

```diff
@@ -7,4 +7,4 @@
     remote:        data-remote
```

Comment:
Can't find any use of these kaminari things - do you have examples? Maybe they're only admin side?

Reply:
Looked into this, and it seems like kaminari is only used for pagination in the admin area. Have reverted these changes as well.